### DISCUSSION TITLE
prettier plugin for tailwind classNames sorting.

### DISCUSSION THREAD

**Discussion by vinaykulk621** [vercel/next.js#47366] [Ideas]
### Goals

1. Addition of prettier plugin for tailwind css.
2. Make our life easy by sorting the class Names of tailwind css 

### Background

As a next js and taiwind css user it is easy to work with prettier plugin which sorts the classnames for me here is the blog post that explains the plugin that sorts the classnames [prettier-plugin-for-tailwindcss](https://tailwindcss.com/blog/automatic-class-sorting-with-prettier)

### Proposal

While running `npx create-next-app` ask the user if they want to use tailwind with prettier plugin.

**Comment by inwardmovement** [comment]
+1
It would install prettier in the meantime, and add `format` and `format:check` npm scripts.
